ibis-server/app/dependencies.py

```
@@ -35,3 +36,12 @@ def _filter_headers(header_string: str) -> bool:
     elif header_string == "sentry-trace":
         return True
     return False
+    return request.headers
```

Ruff (0.8.2)

Undefined name `request`

(F821)